backend/benefit/messages/views.py

```diff
@@ -0,0 +1,53 @@
+from applications.models import Application
+from common.permissions import BFIsAuthenticated, BFIsHandler, TermsOfServiceAccepted
+from django.conf import settings
+from django.utils.translation import gettext_lazy as _
+from messages.models import Message
+from messages.permissions import IsMessageAuthor
+from messages.serializers import MessageSerializer, NoteSerializer
+from rest_framework import viewsets
+from rest_framework.exceptions import NotFound
+from users.utils import get_company_from_request
+
+
+class ApplicantMessageViewSet(viewsets.ModelViewSet):
+
+    serializer_class = MessageSerializer
+    permission_classes = [BFIsAuthenticated, TermsOfServiceAccepted, IsMessageAuthor]
```

Comment:
Since the applicant is operating on behalf of their company, should they see all of their company's messages?